src/components/single-page/single-page-content.tsx

```diff
@@ -23,6 +23,10 @@ export const SinglePageContent: React.FC<IProps> = (props) => {
   const renderPageContent = (page: Page, index: number) => {
     // const visibleEmbeddables: VisibleEmbeddables = getVisibleEmbeddablesOnPage(page);
     // const embeddables = [...visibleEmbeddables.headerBlock, ...visibleEmbeddables.interactiveBox, ...visibleEmbeddables.infoAssessment];
+    const onSizeChange = () => {
```

Comment:
Needed to add this function for code to compile. Currently a place holder until single page content is added back in.